### PR TITLE
[ch1115] Added topic `chat.create-case`

### PR DESCRIPTION
**Pull request by dmarin**

### CODE REVIEW

File: docs/chat.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### AriesSDK.chatCreateCase(properties: Object)
+
+```js readonly
+const caseProperties = {
+    category: 'Big Problem',
+    subcategory: 'whatever'
+    }
```
Comment (jasonadt) [inline docs/chat.md]: (nit) please remove this line (extra closing brace)

File: docs/chat.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+};
+AriesSDK.chatCreateCase(caseProperties);
+```
+
+Initiates the native flow to create a case. `category` and `subcategory` are mandatory properties, native will validate they exists and fail if they are not provided or empty.
```
Comment (jasonadt) [inline docs/chat.md]: (sp) should be: `... will validate that they exist...`

File: src/AriesSDK.js [hunk 3]
```diff
@@ -99,6 +99,8 @@ const verifySelfie = (userId) => publishMessageToBusAndWaitForResponseWithMatchi
 
 const webViewLoaded = () => publishMessageToBusAndWaitForResponseWithMatchingId('view.loaded');
 
+const chatCreateCase = () => publishMessageToBusAndWaitForResponseWithMatchingId('chat.create-case');
```
Comment (jasonadt) [inline src/AriesSDK.js]: Is this line missing the case object arguments?

**Review by jasonadt — CHANGES_REQUESTED** [review]
some minor comments

File: src/AriesSDK.js (hunk 3 above)
Comment (dmarin) [reply src/AriesSDK.js]: indeed!! Good catch! thanks

**Review by jasonadt — APPROVED** [review]
👍  ship it!

**Review by jantocv — APPROVED** [review]
🚀 GREAT 🔥

**Pull request merged**